### PR TITLE
Make docker capable version of Testaframe for easy setup.

### PR DESCRIPTION
**Pull request by Bungalow12**
This adds support for test execution from within a Docker container.
In doing so it changes the default `OSBrowser` to `Local_Chrome`
I also converted the original RST to the Git supported and more widely used Markdown format
It also defaults `Local_Chrome` to full screen.
Then this adds a `Dockerfile` for docker setup plus a `docker-compose.yml` for even easier setup.
Lastly, this adds a shell script to execute tests within the docker container `run_tests.sh` It is literally a light wrapper over `nosetests run_user.py` and would be run using `docker-compose exec app ./run_tests.sh` This way all of the nose parameters can still be used.

See the README for added setup instructions as well as how to access the viewer via a VNC client.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,880 @@
...
+
+``` bash
+cp run_local.py run_user.py
+```
+
+Next, you will need to create a personal `my_cfg.py` on your local machine by
+using `my_cfg_example.py` as an example. Create one using
+
+``` bash
+cp my_cfg_example.py my_cfg.py
```
Comment (emailit2me2) [inline README.md]: needs the config/ in front now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,880 @@
...
+Setting up the Docker environment is simple.
+
+``` bash
+docker-compose up
+```
+
+To run the tests:
+
+``` bash
+docker-compose exec app ./run_tests.sh -s -v -a Wiki
```
Comment (emailit2me2) [inline README.md]: Running this out of the box yielded an error.
  File "/home/seluser/testaframe/config/env.py", line 167, in get_se_driver
    opts = Options()
NameError: global name 'Options' is not defined

File: run_local.py [hunk 3]
```diff
@@ -18,8 +18,8 @@
 ]
 
 browsers = [
-    'Local_FF',
-    #  'Local_Chrome',
+    #  'Local_FF',
```
Comment (emailit2me2) [inline run_local.py]: I switched to Local_FF and it said the gecko driver is missing.  Can we add that to the base docker image.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Set the base image to Ubuntu
+
+FROM selenium/standalone-chrome-debug
+
+# Necessary for HEADLESS
```
Comment (emailit2me2) [inline Dockerfile]: Is this still a valid comment?  If so, what does it mean?

**Review by emailit2me2 — CHANGES_REQUESTED** [review]
Pretty nice to have a reliable environment up in minutes.

File: Dockerfile (hunk 4 above)
Comment (MozJon) [reply Dockerfile]: No I guess it's not :/

File: run_local.py (hunk 3 above)
Comment (MozJon) [reply run_local.py]: Oh fun I must have a pip package installed or something . I'll figure it out as it worked on my box :/

File: README.md (hunk 2 above)
Comment (MozJon) [reply README.md]: Oh fun I must have a pip package installed or something . I'll figure it out as it worked on my box :/

**Review by emailit2me2 — APPROVED** [review]

**Comment by emailit2me2** [comment]
Still some work to do, but Docker + Py3 works.

**Pull request merged**